cnd/src/http_api/action.rs

```diff
@@ -34,13 +37,13 @@ pub enum ActionExecutionParameters {
 #[serde(tag = "type", content = "payload")]
 pub enum ActionResponseBody {
     BitcoinSendAmountToAddress {
-        to: bitcoin_support::Address,
+        to: bitcoin::Address,
         amount: String,
-        network: bitcoin_support::Network,
+        network: bitcoin::Network,
     },
     BitcoinBroadcastSignedTransaction {
         hex: String,
-        network: bitcoin_support::Network,
+        network: bitcoin::Network,
```

Comment:
How does the HTTP API output looks like know? Can you add a test so that it's obvious?